### ISSUE TITLE
Display keyboard shortcuts with `?` e.g. `Shift` + `/`

### ISSUE THREAD

**Issue by Pinjasaur** [silverbulletmd/silverbullet#1]
Would be helpful to display available keyboard shortcuts with <kbd>?</kbd> (<kbd>Shift</kbd> + <kbd>/</kbd>). For example, github.com supports that behavior.

Maybe even a `Help` command as an alias in the command palette?

**Comment by zefhemel** [comment]
Since SB is in always-on edit mode, pressing “?” would always insert a “?” character. We can come up with another shortcut, though. Generally all keyboard shortcuts are also listed in Cmd-/. You think it would be helpful to have a distilled version of commands, just the ones that have keyboard shortcuts associated?

**Comment by Pinjasaur** [comment]
Ah, that's a good point—didn't consider the literal `?`. As a first-time user I was just trying to find a sort of "help page". A random keyboard shortcut isn't very pleasant UX-wise so maybe it would be best to have a `Help` command in the command palette. Could be as simple as a native `alert()` with a few words/sentences.

It's sort of akin to if I'm struggling with a CLI tool: I blindly try `-h` and `--help` to see if I can get anywhere quickly.

**Comment by zefhemel** [comment]
Right, I am definitely thinking of incorporating some sort of help system/documentation inside the app itself. And likely there would be an obvious way to get to it, potentially even a button 🤡
